src/libraries/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -67,6 +67,8 @@
     <Compile Include="System\Reflection\Context\Virtual\VirtualReturnParameter.cs" />
     <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\RequiresAssemblyFilesAttribute.cs"
              Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0'))" />
+    <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\RequiresAssemblyFilesAttribute.cs"
```

Comment:
I think this is extra is just a repetition of the previous include as far as I see

Reply:
Agreed - these 2 includes should be pulled out into an ItemGroup that has a Conditon on it:

```xml
<ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0'))">
  <Compile ...
  <Compile ...
```